tests/e2e/cucumber/features/smoke/admin-settings/groups.ocis.feature

```diff
@@ -5,6 +5,24 @@ Feature: groups management
     And "Admin" opens the "admin-settings" app
     And "Admin" navigates to the groups management page
     When "Admin" creates the following groups
-      | id    |
-      | sales |
+      | id       |
+      | sales    |
+      | security |
```

Comment:
I wanted to add checking  `Then "Admin" should see the following group` here but stuck with setting group uuid  after creating group via UI